.circleci/config.yml

```diff
@@ -1,28 +1,135 @@
-# Javascript Node CircleCI 2.0 configuration file
+# Javascript Node CircleCI 2.1 configuration file
 #
 # Check https://circleci.com/docs/2.0/language-javascript/ for more details
 #
-version: 2
+version: 2.1
+executors:
+  docker-publisher:
+    environment:
+      IMAGE_NAME: italia/developers-italia-onboarding
+    docker:
+      - image: circleci/buildpack-deps:stretch
 jobs:
   build:
     docker:
-      - image: circleci/node:8 
-      
-    working_directory: ~/repo
+      - image: circleci/node:8
```

Comment:
I have a doubt regarding version 8. Checking https://nodejs.org/it/about/releases/ it looks like it's going in EOL on the 31/12. I believe it could be good to start testing 10 LTS, what do you think? This may not be related to this file (which may be OK with 8), let me know if I should open an issue regarding this.

Reply:
As for spec [here](https://circleci.com/docs/2.0/circleci-images/#nodejs) we could use `lts` tag which point the latest stable build.